tests/admin/test_admins.py

```diff
@@ -74,3 +74,103 @@ def test_get_admins_iterator(self):
                 'limit': ['100'],
                 'offset': ['0'],
             })
+
+    def test_get_external_password_mgmt_statuses(self):
```

Comment:
I tried to get nose2 to run these tests in a parameterized way but could not get it to work (it seemed to refuse to run setUp() before each test)